### ISSUE TITLE
Add Start and Stop Functional Buttons on the Homepage

### ISSUE THREAD

**Issue by NebulaTris** [NebulaTris/vibescape#1]
Presently Only Check my emotion button exists,
<img width="536" alt="image" src="https://github.com/NebulaTris/vibescape/assets/94922914/79c72236-ae62-4384-9aad-5f6f6d39f588">
Would like a Start and Stop Button to be implemented instead of check my emotion which are fully functional and allow the model to run correctly.

**Comment by AyushSenDazzler** [comment]
Hi there,

I want clarification about the stop button you wanted to implement. Like, what purpose would it serve in the given context? Is it about controlling emotion detection? Right now, even when the pause button on the video feed is clicked, emotion is still detected by the model in the background.

Thanks!

**Comment by NebulaTris** [comment]
Yes the stop button is to implemented so that emotion detection stops and gets feeded into the variable of emotion.

**Comment by AyushSenDazzler** [comment]
I have understood your codebase. Could you assign me this issue? I'm planning to add a new button, "Stop," which will change webrtc_streamer's default_playing_state from "TRUE" to "FALSE." Subsequently, the EmotionProcessor will stop as no frames will get into it. To remove ambiguity, I'm using another session state: emotion.

**Comment by NebulaTris** [comment]
Sure, Sounds great
